tempodb/encoding/vparquet/wal_block.go

```diff
@@ -514,6 +514,16 @@ func (b *walBlock) SearchTags(ctx context.Context, cb common.TagCallback, opts c
 }
 
 func (b *walBlock) SearchTagValues(ctx context.Context, tag string, cb common.TagCallback, opts common.SearchOptions) error {
+	// Wrap to v2-style
+	cb2 := func(v traceql.Static) bool {
+		cb(strings.Trim(v.String(), "`"))
```

Comment:
any particular reason we're stripping backticks?

Reply:
This is wrapper that casts the new v2 `traceql.Static` back to v1 `string`.  Static stringer [includes backticks](https://github.com/grafana/tempo/blob/main/pkg/traceql/ast_stringer.go#L67) for AST roundtrip printing.  This seemed better than reproducing the switch statement, but open to other possibilities.  This is also exists in `backendBlock.SearchTagValues`

Reply:
Good point.  Added a helper method `Static.EncodeToString(quotes bool)` and refactored.